components/markets/MarketScroll.tsx

```diff
@@ -1,38 +1,48 @@
-import { motion } from "framer-motion";
-import { useStore } from "lib/stores/Store";
 import { observer } from "mobx-react";
-import Link from "next/link";
 import { useEffect, useRef, useState } from "react";
-import { ChevronLeft, ChevronRight } from "react-feather";
 import { useResizeDetector } from "react-resize-detector";
-import MarketCard, { IndexedMarketCardData } from "./market-card";
+import MarketCard, { IndexedMarketCardData } from "./market-card/index";
+import HorizontalScroll from "components/ui/HorizontalScroll";
 
 const MarketScroll = observer(
   ({
     title,
     markets,
-    showMarketsLink = true,
+    link,
   }: {
     title: string;
     markets: IndexedMarketCardData[];
-    showMarketsLink?: boolean;
+    link?: string;
   }) => {
     const scrollRef = useRef<HTMLDivElement>();
     const [scrollLeft, setScrollLeft] = useState(0);
     const [scrollDirection, setScrollDirection] = useState<"left" | "right">(
       "right",
     );
     const { width: containerWidth, ref: containerRef } = useResizeDetector();
-    const cardWidth = 320;
-    const gap = 30;
+    const gap = 16;
+    //calculate cards shown and width based on container width
+    const cardsShown =
+      containerWidth < 716
+        ? 3
```

Comment:
Think this is a typo. on small screens it shows only one card but scrolls 3 at a time.

Reply:
I made a mistake on the scroll functionality breakpoint; scrolling shouldn't appear until 905px breakpoint. I have made the adjustment now.

The market cards should stack one-by-one until hitting 905px so the logic on cardsShown for < 716 is unnecessary; I have removed it.